test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -135,6 +147,12 @@ tests rdb = testGroup "Chainweb.Test.Pact.RemotePactTest"
         withResource' getCurrentTimeIntegral $ \(iotm :: IO (Time Micros)) ->
             let cenv = _getServiceClientEnv <$> net
                 iot = toTxCreationTime <$> iotm
+                pactDir = do
+                  m <- _getNodeDbDirs <$> net
+                  case M.lookup 0 m of
```

Comment:
This looks up the pactDbDir for node 0. This is kind of a hack, because there is only one node in this test. However, it doesn't matter much, because we are dealing with both submitting `/local` txs and compaction, so picking an arbitrary node to run these two operations on is fine.

Reply:
I have no idea. I think this test suite (RemotePactTest and some others) were originally meant to be flexible enough to run multiple nodes, but they don't. AFAIK only the slow-tests (MultiNode.hs) actually run multiple nodes.

I don't want to make the change in this PR though.